examples/nirfmxspecan/spectrum-basic.py

```diff
@@ -0,0 +1,151 @@
+# Link: https://github.com/ni/grpc-device/wiki/NI-RFmxSpecAn-C-Function-Reference
+#
+# Running from command line:
+#
+# Server machine's IP address, port number, and physical channel name can be passed as separate command line arguments.
+#   > python spectrum-basic.py <server_address> <port_number> <physical_channel_name>
+# If they are not passed in as command line arguments, then by default the server address will be "localhost:31763", with "SimulatedRFSA" as the resource name
+
+from math import floor
+import grpc
+import sys
+
+import nirfmxspecan_pb2 as nirfmxspecan_types
+import nirfmxspecan_pb2_grpc as grpc_nirfmxspecan
+
+server_address = "localhost"
+server_port = "31763"
+session_name = "NI-RFSASession"
+
+# Resource name and options for a simulated 5663 client.
+resource = "SimulatedRFSA"
+options = "Simulate=1,DriverSetup=Model:5663"
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    server_address = sys.argv[1]
+if len(sys.argv) >= 3:
+    server_port = sys.argv[2]
+if len(sys.argv) >= 4:
+    resource = sys.argv[3]
+    options = ""
+
+# Create a gRPC channel + client.
+channel = grpc.insecure_channel(f"{server_address}:{server_port}")
+client = grpc_nirfmxspecan.NiRFmxSpecAnStub(channel)
+instr = None
+
+
+def format_frequency(f):
+    # adapted from https://stackoverflow.com/a/1094933
+    suffix = "Hz"
+    for unit in ["", "k", "M", "G"]:
+        if abs(f) < 1000.0:
+            return f"{f:3.3f} {unit}{suffix}"
+        f /= 1000.0
+    return f"{f:.3f} G{suffix}"
+
+
+# Raise an exception if an error was returned
+def raise_if_error(response):
+    if response.status != 0:
+        error_response = client.GetError(
+            nirfmxspecan_types.GetErrorRequest(
+                instrument_handle=instr,
+            )
+        )
+        raise Exception(f"Error: {error_response.error_description}")
+
+    return response
+
+
+try:
+    init_response = raise_if_error(
+        client.Initialize(
+            nirfmxspecan_types.InitializeRequest(
+                session_name=session_name, resource_name=resource, option_string=options
+            )
+        )
+    )
+    instr = init_response.handle_out
+
+    raise_if_error(
+        client.CfgRF(
+            nirfmxspecan_types.CfgRFRequest(
+                instrument_handle=instr, selector_string="",
+                center_frequency=1e9, reference_level=0, external_attenuation=0
+            )
+        )
+    )
+
+    raise_if_error(
+        client.SpectrumCfgSpan(
+            nirfmxspecan_types.SpectrumCfgSpanRequest(
+                instrument_handle=instr, selector_string="", span=1e6
+            )
+        )
+    )
+
+    raise_if_error(
+        client.SpectrumCfgRBWFilter(
+            nirfmxspecan_types.SpectrumCfgRBWFilterRequest(
+                instrument_handle=instr, selector_string="",
+                rbw_auto=nirfmxspecan_types.SPECTRUM_RBW_AUTO_BANDWIDTH_TRUE,
+                rb_w=100e3,
```

Comment:
(Not necessarily for this PR): Looks like we need to fix this param name. Not sure what happens when this goes through the various transformations. But I think the fix is probably: "RBW" -> "rbw" in the metadata.

Reply:
Yeah, the metadata has the parameter as "rbW". Filed #436 for this.